hack/bundler.sh

```diff
@@ -2,4 +2,5 @@
 set -euxo pipefail
 cd route-monitor-operator
 make packagemanifests
-chmod 775 -R packagemanifests
+NEW_VERSION=$(ls "$BUNDLE_DIR" | sort -t . -k 3 -g | tail -n 1)
```

Comment:
This is good solution, but a better one would be taking the prev version from the yaml file in the saas.../rmo/*.yaml

Reply:
this is not a good solution, it's a hack as the whole script.

Reply:
The other way would be
```
grep currentCSV route-monitor-operator.package.yaml | sed 's/.*v'//g
```
but it doesn't look much better. Since my original version is the way we do it in the rest of the script, I'd prefer to leave it as is.